va/va.go

```diff
@@ -490,6 +499,13 @@ func (va VAImpl) fetchHTTP(identifier string, token string) ([]byte, string, *ac
 	httpRequest.Header.Set("User-Agent", userAgent())
 	httpRequest.Header.Set("Accept", "*/*")
 
+	addrs, err := va.resolveIP(identifier)
+
+	if err != nil || len(addrs) == 0 {
```

Comment:
Let's break these out into two separate cases, so we can pass through the error from `resolveIP`.

Reply:
Done.